invenio_communities/assets/semantic-ui/js/invenio_communities/administration/config.js

```diff
@@ -0,0 +1,43 @@
+// This file is part of Invenio-requests
+// Copyright (C) 2022 CERN.
+//
+// Invenio-requests is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import axios from "axios";
+
+const apiConfig = {
+  withCredentials: true,
+  xsrfCookieName: "csrftoken",
+  xsrfHeaderName: "X-CSRFToken",
+  headers: {
+    "Accept": "application/vnd.inveniordm.v1+json",
+    "Content-Type": "application/json",
+  },
+};
+
+export const http = axios.create(apiConfig);
```

Comment:
Is this suppose to use the central axios from react-invenio-forms?

Reply:
not integrated yet, therefore I didn't use it